solver/pb/ops.proto

```diff
@@ -64,6 +64,7 @@ message Meta {
 	repeated Ulimit ulimit = 9;
 	string cgroupParent = 10;
 	bool removeMountStubsRecursive = 11;
+	repeated SecretEnv secretenv = 12;
```

Comment:
This looks like a duplication with `ExecOp` . I think it should be in `gateway.InitMessage`